proctor-consumer/src/main/java/com/indeed/proctor/consumer/ProctorConsumerUtils.java

```diff
@@ -76,14 +79,14 @@ public static ProctorResult determineBuckets(
      * @return a map of test names to bucket values specified by the request.  Returns an empty {@link Map} if nothing was specified
      */
     @Nonnull
-    public static ForceGroupsOptions parseForcedGroupsOptions(@Nonnull final HttpServletRequest request) {
+    public static ForceGroupsOptions parseForcedGroupsOptions(@Nonnull final HttpServletRequest request, final Set<String> forcePayloadTests) {
```

Comment:
There are a few users of this method in addition to proctor library itself (https://link.indeed.tech/GLYDDM3NZ), and those users will get a compile error by this (not backward compatible) change.

My suggestion is to keep maintaining `public static ForceGroupsOptions parseForcedGroupsOptions(@Nonnull final HttpServletRequest request)` (in addition to the new signature) and make the old signature method call this with empty `forcePayloadTests`.

Reply:
Should ProctorConsumerUtils.determineBuckets() also have the old parameters maintained as a separate function?